Java/TelemetrySystem/src/main/java/tddmicroexercises/telemetrysystem/TelemetryOperations.java

```diff
@@ -0,0 +1,40 @@
+package tddmicroexercises.telemetrysystem;
+
+public class TelemetryOperations implements TelemetryOpsInterface {
+
+
+    private final String DiagnosticChannelConnectionString = "*111#";
+
+    TelemetryDiagnosticInterface telemetryDiagnostic = new TelementryDiagnosticControlsImpl();
```

Comment:
This is hard coupling
Lets initialise this by passing object in constructor params